api/routes/issues.js

```diff
@@ -0,0 +1,185 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+
+const Issue = require('../models/issue');
+const Comment = require('../models/comment');
+
+// GET / - list all created issues
+router.get('/', (req, res, next) => {
+    Issue.find()
+        .select('-__v')
+        .exec()
+        .then(docs => {
```

Comment:
Variables should be named so they describe what they are. In this case `docs` could lead reader to think there is a `Doc` entity. How about name this variable `issues`?